multus/multus.go

```diff
@@ -190,6 +193,16 @@ func cmdAdd(args *skel.CmdArgs, exec invoke.Exec, kubeClient k8s.KubeClient) (cn
 		if delegate.MasterPlugin || result == nil {
 			result = tmpResult
 		}
+
+		//create the network status, only in case Multus as kubeconfig
+		if n.Kubeconfig != "" {
+			delegateNetStatus, err := types.LoadNetworkStatus(tmpResult, delegate.Name, delegate.MasterPlugin)
+			if err != nil {
+				return nil, fmt.Errorf("Multus: Err rethe networks status: %v", err)
+			}
+
+			n.AddNetworkStatus(delegateNetStatus)
```

Comment:
Instead of adding a field to our NetConf, since all this is in cmdAdd, would it be simpler to just declare `var netStatus    []*NetworkStatus` at the top of cmdAdd(), and append each status to it, then check later for `if len(netStatus) > 0 { <update pod status>`?

Reply:
yes, I got it .. I am using `AddNetworkStatus` only in the `cmdAdd()` will include the netstatus length checking along with `kubeconfig`